spec/jsonapi_errors_handler_spec.rb

```diff
@@ -1,11 +1,91 @@
 require 'spec_helper'
 
-RSpec.describe JsonapiErrorsHandler do
-  it "has a version number" do
-    expect(JsonapiErrorsHandler::VERSION).not_to be nil
+class TestJsonapiErrorsHandler
+  include JsonapiErrorsHandler
+  
+  def render(json: {}, status:)
+    json.to_h.merge(status: status)
   end
+end
+
+RSpec.describe JsonapiErrorsHandler do 
+  let(:dummy) { TestJsonapiErrorsHandler.new }
+  
+  it 'expects ErrorMapper to have already mapped some errors' do
+    dummy
```

Comment:
that's not needed I believe. You `map_errors!` at the moment the class is defined (in the included section). Not when an instance of this class is created. Without this line it should also pass.